tests/explainability/test_explainable_user_model_tag_based.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from moviebot.explainability.explainable_user_model_tag_based import (
+    ExplainableUserModelTagBased,
+)
+
+
+@pytest.fixture
+def explainable_model():
+    return ExplainableUserModelTagBased(
+        "moviebot/explainability/explanation_templates.yaml"
```

Comment:
I am wondering if it you be best to make `_DEFAULT_TEMPLATE_FILE` public and reuse it here?

Reply:
Done